src/test/java/elements/FreeShippingCheckBox.java

```diff
@@ -0,0 +1,40 @@
+package elements;
+
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FreeShippingCheckBox {
+	public static final String FREESHIPPING = "Free international shipping";
+	private static final String FREESHIPPING_LOCATOR = "//div[@id='LeftNavContainer']//span[contains(text(),'" + FREESHIPPING + "')]";
+	private static final String ITEMS_FREESHIPPING_LOCATOR = "//div[@id='Results']//span[contains(text(),'"+ FREESHIPPING +"')]";
+	private WebDriver driver;
+	@FindBy(xpath = FREESHIPPING_LOCATOR)
+	private WebElement freeShipingCheck;
+	@FindBy(xpath = ITEMS_FREESHIPPING_LOCATOR)
```

Comment:
хмм, может я что-то неправильно понял но в доках написано:(@FindAll) It will then search for all elements that match any of the FindBy criteria, то есть его используют, когда нужно найти хотя бы любой из элементов? Или тут есть нюансы?